### DISCUSSION TITLE
¿Qué clase de habilidades puedo adquirir en este curso que son demandadas en el mercado laboral actual?

### DISCUSSION THREAD

**Discussion by unnusual** [hackatomik/SISCOx256#11] [Preguntas Profesionales | Carrera]
Me gustaría entender cómo lo que aprenda en este curso puede impactar mi carrera profesional. Quisiera saber si las herramientas y conocimientos que vamos a desarrollar están alineados con lo que las empresas están utilizando actualmente. Si es así, ¿podrían dar un ejemplo concreto?

**Comment by tapilew** [comment]
Toda empresa que maneja software debe usar git y GitHub... o sino Mercurial en caso de Facebook/Meta, pero también se trata de control de versiones a fin de cuentas. Es la forma más cómoda de trabajar en equipo y realizar cambios en tu trabajo de forma segura y ordenada.

Git es el sistema de control de versiones con la línea de comando. GitHub es este sitio web que almacena código usando Git y permite colaborar remotamente. Tiene características adicionales tal como estos foros de discusión o llevar el registro de bugs o tareas con GitHub Projects. Con esa característica de GitHub es posible implementar "marcos de trabajo ágiles" como Scrum, una forma muy común en la mayoría de empresas para gestionar proyectos y administrar equipos.

**Comment by tapilew** [comment]
¿Qué carrera cursas y qué expectativas tienes de ella? ¿Tienes algún rol laboral en mente? Me sería útil esa info para darte recomendaciones más específicas a tu caso personal.

**Comment by JamSwiftDev** [comment]
mira para responder a tu pregunta debes ser primero a que rama te vas a dedicar porque si es en software yo te puedo asesorar de forma completa, pero si es hardware mi conocimiento no es completo o si es cyber seguridad tengo un conocimiento medio completo, asi que pediria que nos dieras de forma mas detalla que rama quieres tomar que quieres trabajar, porque mi objetivo siempre es ese asesorarlos de una forma completa para que puedan ejercer por eso les proporcione mi numero de whattsap para que me contacten y yo puedo sacar un tiempo ya se una hora de clase por meets para eso.

**Comment by quantumquirkxyz** [comment]
En un curso que aborde áreas como Git, GitHub, automatización, y herramientas relacionadas con la inteligencia artificial y el desarrollo de software, puedes adquirir habilidades altamente demandadas en el mercado laboral actual. Aprender a usar Git y GitHub te capacita para gestionar versiones de código y colaborar eficazmente en proyectos, una habilidad esencial en empresas tecnológicas como Google y Microsoft. La automatización de tareas mejora la eficiencia y reduce errores, algo crucial en roles de DevOps en compañías como Amazon y Netflix. Además, el conocimiento en infraestructura como código (IaC) es valioso para gestionar entornos de nube de manera controlada, tal como se hace en IBM y Red Hat. Manejar y analizar datos, desarrollar aplicaciones y APIs, y aplicar prácticas de seguridad y privacidad son competencias cada vez más requeridas en empresas como Facebook, Stripe y Palo Alto Networks. Estas habilidades te prepararán para enfrentar los desafíos tecnológicos actuales y te harán un candidato competitivo en diversos campos tecnológicos.
